src/main/java/de/tum/in/www1/artemis/web/rest/dto/ExamScoresDTO.java

```diff
@@ -0,0 +1,102 @@
+package de.tum.in.www1.artemis.web.rest.dto;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
```

Comment:
Should we use @JsonInclude(JsonInclude.Include.NON_EMPTY) like with the other entities. I'm not sure what the client expects.

Reply:
yes